### PR TITLE
Changing the global typography elementary tokens

### PR DESCRIPTION
**Pull request by StefanIvanov**

### CODE REVIEW

File: data/tokens.json [hunk 1]
```diff
@@ -263,6 +267,27 @@
...
+          "x": "0",
+          "y": "0",
+          "blur": "0",
+          "spread": "3",
+          "color": "{Primary.gtmds_color_primary-color-focus}",
+          "type": "dropShadow"
+        }
+      ],
+      "type": "boxShadow"
+    },
```
Comment (StefanIvanov) [inline data/tokens.json:290]:
```suggestion
```

File: data/tokens.json [hunk 2]
```diff
@@ -32,6 +32,10 @@
       "value": "#fffae7",
       "type": "color"
     },
+    "gtmds_color_red-12": {
+      "value": "#913042",
+      "type": "color"
+    },
```
Comment (StefanIvanov) [inline data/tokens.json:38]:
```suggestion
```

**Review by Ddungyov — APPROVED** [review]
Splendid, just md and sm

**Comment by StefanIvanov** [comment]
> Splendid, just md and sm

Now it is even splendid-er after it is not just md and sm but even reduced to a singular value and components are updated accordingly. Sassy will be the final stop on Friday and we will be ready to merge this avalanche of changes

**Comment by Ddungyov** [comment]
> 

I am just curious to see if it will damage our design after updating the library. This should be tested

**Comment by StefanIvanov** [comment]
> > 
> 
> I am just curious to see if it will damage our design after updating the library. This should be tested

Not really damage but there we are few necessary changes that I have of course tested in the typography branch already https://www.figma.com/file/Uopb68lCUHeN4rQ9qKetLb/branch/MS6nlQJErP7tgiMFG6GJf4/Design-System-3.0

**Pull request merged**